### ISSUE TITLE
MFA Login to Azure SQL Databases

### ISSUE THREAD

**Issue by atfernando** [Microsoft/sqlopsstudio#1847]
- SQL Operations Studio Version:

Steps to Reproduce:

Please add the ability to login to Azure SQL databases with MFA ebabled accounts.

**Comment by kevcunnane** [comment]
This is a duplicate of #13. We're currently waiting on .Net Core support for AAD authentication, which I believe is due to be worked on soon. Once this gets done, it'll be possible to look at this feature in SQL Operations Studio. I'll close this so we just have a single issue tracking the request - please follow #13 for further updates about this.
